public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/AddAnomalyDetector.tsx

```diff
@@ -0,0 +1,823 @@
+      },
+    };
+  }
+
+  function featureListToFormik(featureList): FeaturesFormikValues[] {
+    return featureList.map((feature) => {
+      return {
+        featureId: feature.id,
+        featureName: getFeatureNameFromParams(feature.id),
+        featureEnabled: true,
+        featureType: FEATURE_TYPE.SIMPLE,
+        aggregationBy: 'avg',
+        aggregationOf: [{ label: feature.params.field.name }],
+        aggregationQuery: formikToAggregation(feature),
+      };
+    });
+  }
+
+  return (
+    <div className="add-anomaly-detector">
+      <Formik
+        initialValues={initialDetectorValue}
+        onSubmit={handleSubmit}
+        validateOnChange={false}
+      >
+        {(formikProps) => (
+          <>
+            <EuiFlyoutHeader hasBorder>
+              <EuiTitle>
+                <h2 id="add-anomaly-detector__title">Add anomaly detector</h2>
+              </EuiTitle>
+            </EuiFlyoutHeader>
+            <EuiFlyoutBody>
+              <div className="add-anomaly-detector__scroll">
+                <EuiFormFieldset
+                  legend={{
+                    display: 'hidden',
+                    children: (
+                      <EuiTitle>
+                        <span>
+                          Options to create a new detector or associate an
+                          existing detector
+                        </span>
+                      </EuiTitle>
+                    ),
+                  }}
+                  className="add-anomaly-detector__modes"
+                >
+                  {[
+                    {
+                      id: 'add-anomaly-detector__create',
+                      label: 'Create new detector',
+                      value: 'create',
+                    },
+                    {
+                      id: 'add-anomaly-detector__existing',
+                      label: 'Associate existing detector',
+                      value: 'existing',
+                    },
+                  ].map((option) => (
+                    <EuiCheckableCard
+                      {...{
+                        ...option,
+                        key: option.id,
+                        name: option.id,
+                        checked: option.value === mode,
+                        onChange: () => setMode(option.value),
+                      }}
+                    />
+                  ))}
+                </EuiFormFieldset>
+                <EuiSpacer size="m" />
+                {mode === 'create' && (
+                  <div className="create-new">
+                    <EuiText size="xs">
+                      <p>
+                        Create and configure an anomaly detector to
+                        automatically detect anomalies in your data and to view
+                        real-time results on the visualization.{' '}
+                        <a
+                          href="https://opensearch.org/docs/latest/observing-your-data/ad/index/"
```

Comment:
Can use existing constants for this https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/blob/main/public/utils/constants.ts#L54

Reply:
updated with using link constant